easybuild/easyconfigs/h/HPCX/HPCX_OMPI-4.0.x-foss-2018b.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'HPCX_OMPI'
+version = '4.0.x'
+
+homepage = 'http://www.open-mpi.org/'
+description = """Mellanox flavored Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://download.open-mpi.org/release/open-mpi/v4.0']
+sources = [{
```

Comment:
If you're picking this up from OpenMPI directly then it is not the Mellanox version and it is just a normal OpenMPI
And OpenMPI belongs at the GCC level not foss.